pkg/capabilities/triggers/mercury_trigger.go

```diff
@@ -26,9 +25,22 @@ const defaultTickerResolutionMs = 1000
 
 // TODO pending capabilities configuration implementation - this should be configurable with a sensible default
 const defaultSendChannelBufferSize = 1000
+type mercuryTriggerConfig struct {
+	// strings should be hex-encoded 32-byte values, prefixed with "0x", all lowercase, minimum 1 item
+	FeedIDs []string `json:"feedIds" jsonschema:"pattern=^0x[0-9a-f]{64}$,minItems=1"`
+	// must be greater than 0
+	MaxFrequencyMs int `json:"maxFrequencyMs" jsonschema:"minimum=1"`
+}
+
+type mercuryTriggerInputs struct {
+	TriggerID string `json:"triggerId"`
+}
+
+var mercuryTriggerValidator = capabilities.NewValidator[mercuryTriggerConfig, mercuryTriggerInputs, capabilities.TriggerEvent](capabilities.ValidatorArgs{Info: mercuryInfo})
```

Comment:
Why create it here and not inside NewMercuryTriggerService?

Reply:
I was originally writing tests that used just the validator. I don't think I have any here. I can either inline it or keep the pattern consistent. I thought it may be useful when we have complex capabilities where we dont want to bother settting up the capability just to test validation